frontend/src/components/special/Subscribe.tsx

```diff
@@ -0,0 +1,94 @@
+import React from 'react';
+import styled from 'styled-components';
+
+const Content = styled.div`
+  margin: 40px;
+
+  label {
+    display: block;
+    font-family: ${({ theme }) => theme.font};
+    font-size: ${({ theme }) => theme.fontSize.default};
+    color: ${({ theme }) => theme.colors.text};
+    margin: 2px;
+  }
+  
+  .email {
+    display: inline-block;
+    text-align: left;
+    width: 16rem;
+    height: 3rem;
+    box-sizing: border-box;
+    font-family: ${({ theme }) => theme.font};
+    font-size: ${({ theme }) => theme.fontSize.default};
+    color: ${({ theme }) => theme.colors.text};
+    border: 2px solid ${({ theme }) => theme.colors.blue};
+    border-radius: 5px;
+    padding: 5px;
+    margin: 5px;
+  }
+  
+  .button {
+    font-family: ${({ theme }) => theme.font};
+    font-size: ${({ theme }) => theme.fontSize.mediumLarge};
+    background: ${({ theme }) => theme.colors.gradients.blue};
+    color: ${({ theme }) => theme.colors.white};
+    width: 16rem;
+    height: 3rem;
+    margin: 5px;
+    box-sizing: border-box;
+    box-shadow: 0 1px 8px rgba(0, 0, 0, 0.24);
+    border: none !important;
+    border-radius: 5px;
+    cursor: pointer;
+  
+    &:hover {
+      cursor: pointer;
+      box-shadow: 0 2px 12px rgba(0, 0, 0, 0.24);
+    }
+    
+    &:focus {
+      outline: none;
+    }
+  }
+`;
+
+function Subscribe() {
+  return (
+    <Content>
+      {/* Begin Mailchimp Signup Form */}
```

Comment:
Yeah I think this would be good to revisit when we do the entire sight revamp for smaller widths - when this occurs the podium and scoreboard are also completely messed up lol so there's def a lot of work to do